packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -21,6 +21,14 @@ const registerElm = document.querySelector('#registration-register');
 const unregisterElm = document.querySelector('#registration-unregister');
 const registrationStatusElm = document.querySelector('#registration-status');
 
+// Disable screenshare on join in Safari patch
+const isSafari = !!navigator.userAgent.match(/Version\/[\d.]+.*Safari/);
```

Comment:
Can you make the regex testing consistent?
```suggestion
const isSafari = /Version\/[\d.]+.*Safari/.test(navigator.userAgent);
```

Reply:
I just grabbed this off stackoverflow, so I’ll update it